chrome/content/zotero/xpcom/itemPaneManager.js

```diff
@@ -55,7 +57,7 @@
  * A `refresh` is exposed to plugins to allows plugins to refresh the section when necessary,
  * e.g. item modify notifier callback. Note that calling `refresh` during initialization
  * have no effect.
- * @typedef  {SectionHookArgs & { newData: { type: "item" | "mode" | "inTrash" | "tabType", value: any } }} SectionDataChangeHookArgs
+ * @typedef  {SectionHookArgs & { newData: Zotero.Item }} SectionDataChangeHookArgs
```

Comment:
This shouldn't be `newData` if we're just passing an item now. Why wouldn't `onItemChange` just pass the new item as `item`?

Reply:
I see we're passing the old item as `item`:

> Zotero.debug(`Section item data changed from ${item?.id} to ${newData?.id}`);

I'm not sure if that's necessary, but if so, I feel like it should be `oldItem` and `item` or something. But also, the old item might not even exist anymore — it could've been deleted. Do we really need to pass it?

Reply:
```javascript
onRender: (props) => {
    let { body, item, setL10nArgs, setSectionSummary, setSectionButtonStatus } = props;
```

Also, while it's not in this code, for the examples we can just use destructuring assignment right in the parameters line:

```javascript
onRender: ({ body, item, setL10nArgs, setSectionSummary, setSectionButtonStatus }) => {
```

No need to define `props` at all.

Reply:
done